javascript/src/main/pages/History/Ge.js

```diff
@@ -24,13 +26,18 @@ const Ge = () => {
     //const [courseJSON, setCourseJSON] = useState('{"course" : "cs148"}');
     // const courseJSON = '{"course" : "cs156"}';
     const [courseJSON, setCourseJSON] = useState(initialCourseJSON);
-
+    const courseHeaders = [
+        { label: "courseId", key: "courseId" },
+        { label: "title", key: "title" },
+        { label: "units", key: "unitsFixed" }
+    ]
```

Comment:
Should it download all the displayed columns of the search? The deployment doesn't seem to do that.